app/Http/Controllers/StandController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Exceptions\Stand\StandAlreadyAssignedException;
+use App\Exceptions\Stand\StandNotFoundException;
+use App\Models\Stand\Stand;
+use App\Rules\VatsimCallsign;
+use App\Services\NetworkDataService;
+use App\Services\StandService;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class StandController extends BaseController
+{
+    /**
+     * @var StandService
+     */
+    private $standService;
+
+    public function __construct(StandService $standService)
+    {
+        $this->standService = $standService;
+    }
+
+    public function getStandsDependency(): JsonResponse
+    {
+        return response()->json($this->standService->getStandsDependency());
+    }
+
+    public function getStandAssignments(): JsonResponse
+    {
+        return response()->json($this->standService->getStandAssignments());
+    }
+
+    public function createStandAssignment(Request $request): JsonResponse
+    {
+        $invalidRequest = $this->checkForSuppliedData(
+            $request,
+            [
+                'callsign' => ['string' , 'required', new VatsimCallsign],
+                'stand_id' => 'integer|required',
+            ]
+        );
+
+        if ($invalidRequest) {
+            return $invalidRequest;
+        }
+
+        $response = null;
+        try {
```

Comment:
Can you return the responses directly in the try / catch loops respectively?

Reply:
I can but then SonarCloud points out a code complexity violation for too many return statements so went for this one instead. I think I've removed a return now, so will see what I can do!